tracpro/polls/forms.py

```diff
@@ -69,7 +69,7 @@ class ChartFilterForm(forms.Form):
         ('90-days', _("Last 90 days")),
         ('6-months', _("Last 6 months")),
         ('12-months', _("Last 12 months")),
-        ('other', _("Custom range...")),
+        ('custom', _("Custom range...")),
```

Comment:
js needs to be updated to reflect this change from 'other' --> 'custom'

https://github.com/rapidpro/tracpro/blob/develop/tracpro/static/js/poll-charts.js#L112